src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -94,7 +94,7 @@ export const MenuDropdown: FC<MenuDropdownProps> = ({
   largeListCeiling = 0,
   largeListSearch = false,
   searchText = 'Search Accounts',
-  menuTheme = DropdownMenuTheme.Onyx,
+  menuTheme = DropdownMenuTheme.None,
```

Comment:
Instead of the Onyx background, we can change to none and then add the checkmark that matches the figma design for this component.

Reply:
<img width="284" alt="Screen Shot 2022-06-25 at 11 28 41 AM" src="https://user-images.githubusercontent.com/18511823/175786121-520efd54-6468-4a66-917e-02a9b0c49c57.png">